### ISSUE TITLE
Repo under the jqlang organisation? 

### ISSUE THREAD

**Issue by wader** [01mf02/jq-lang-spec#1]
Hello, i think @nicowilliams have had ideas about a language specification repo under the jqlang organisation. Could that be interesting? maybe want to mess around in this repo for a while and then see?

I've also thought about that it would be great with some kind of implementation independent test suite. Would that fit in such a repo?

**Comment by nicowilliams** [comment]
Yes, we do have a repo for this, it's just it's currently empty.  We'd be happy to invite jaq/gojq/jqjq/etc. implementors to be committers/maintainers of a spec repo!  Having an implementation-independent test-suite sounds great!

**Comment by 01mf02** [comment]
Hi @wader & @nicowilliams! I am aware of the remark on the jqlang organisation page about `jqlang/spec`.
At the moment, my specification is in a big flux, but indeed, I would be very happy if this could become the `jqlang/spec` repository one day. My plan is to get this into shape until end of February, at which point I plan to submit this to [some conference](https://icfp24.sigplan.org/track/icfp-2024-papers#Call-for-Papers). That might be a good time for you to decide whether this may become `jqlang/spec`.
Note that the specification will unfortunately diverge in a few parts from the current jq behaviour in order to allow for simpler reasoning about jq programs and for higher execution performance (in particular in the area of updates), but I try to document these divergences.

I also think that an implementation-independent test suite is a good idea. However, I'm not sure whether this repository is an appropriate place for such a test suite. My dream would be an implementation-independent version of the [jq manual](https://jqlang.github.io/jq/manual/) where the examples are automatically tested for several jq implementations. There, I clearly see a place for such an implementation-independent test suite.

**Comment by wader** [comment]
@01mf02 Sounds great! So your working some on a paper that is partially a specification and a presentation for conferences? good luck and i'm eagerly awaiting the result, and don't hesitate asking me for help if you think i can assist with something.

**Comment by 01mf02** [comment]
@wader, yes, I worked on the document such that it can be made into a conference paper as well as a "non-scientific" specification.

From my perspective, I think that this repository would be ready for inclusion into the jq organisation as `jq/spec`, for example. What do you think about it?
